ui/components/MeshConfigComponent.js

```diff
@@ -351,6 +357,12 @@ function MesherySettingsNew({ classes, enqueueSnackbar, closeSnackbar, updatePro
         ), });
       stateUpdater(operatorSwitch, setOperatorSwitch, !operatorSwitch[index], index);
     }, variables);
+    fetchMesheryOperatorStatus({ k8scontextID : contexts[index].id }).subscribe({
+      next : (res) => {
+        setOperatorState(res, index);
+      },
+      error : (err) => console.log("error at operator scan: " + err),
+    });
```

Comment:
You are creating an another subscription here???

The subscription means that once you change the operator state, the subscription created will automatically push you the changed data.

Reply:
yes, I misunderstood, them as queries as in function query is written, got it thankyou @Abhishek-kumar09